### PR TITLE
React capstone project - Metrics webApp

### PR DESCRIPTION
**Pull request by amanuelbelete95**
## List Of Completed Activity
# Core requirements
- Selected  an API to base the development of the web app
- When the page loads, the web app shows the list of categories that some parameter could filter out
- In the details page, the web app retrieves data from the API to show detailed data in the category.
- When the user clicks on (< button) the user navigates to the home page.
- Tested the Project
# Technical requirements
- Created  A redux store for global state management of the project
- The project is a single-page application (SPA) built with React and Redux.
- The data retrieved from the API should is stored in the Redux store.
- Filtered the data retrieved from the API using a Filter stateless component.
- Every page (the main page and the pages for each item) should have a unique route within the SPA.
- The project should be deployed and accessible online using [Render]
- Added a link to the online version of my app and the project description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,212 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank microverse and my coding partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (CollinsTatang) [inline README.md]: - [ ] Please give appropriate credit to  [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/test/HomePage.test.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    const tree = render(
+      <MemoryRouter>
+        <Provider store={store}>
+          <HomePage />
+        </Provider>
+      </MemoryRouter>,
+    );
+    expect(tree).toMatchSnapshot();
+  });
+});
```
Comment (CollinsTatang) [inline src/test/HomePage.test.js:29]: - [ ] Please your unit test components fail all the tests. Kindly correct it and write more tests for several components and pure functions in the Redux code.
test

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @amanuelbelete95, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by abdulhamiid — APPROVED** [review]
Hi @amanuelbelete95 👋 , 

image

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**